GameJamTestProject/Assets/Scripts/BallController.cs

```diff
@@ -7,15 +7,21 @@ public class BallController : MonoBehaviour {
 	public float maxSpeed;
 	public float initSpeed;
 	public Rigidbody2D rb;
+
+    public float rangeMin;
+    public float rangeMax;
 	
-	private float currentSpeed;	
+	private float currentSpeed;
+
+    private Vector2 initVector;
 
 	// Use this for initialization
 	void Start () {
 		rb = GetComponent<Rigidbody2D>();
 		currentSpeed = initSpeed;
-		
-		rb.AddForce(Vector2.one * currentSpeed, ForceMode2D.Impulse);
+        rangeMin = -.75f;
+        rangeMax = .75f;
```

Comment:
Rename to something a little more descriptive

Reply:
ligMin and ligMa(x)